dart/lib/src/sentry_tracer.dart

```diff
@@ -253,6 +253,9 @@ class SentryTracer extends ISentrySpan {
     _measurements.addAll(measurements);
   }
 
+  @visibleForTesting
+  List<SentryMeasurement> get measurements => _measurements;
```

Comment:
SentryTracer calls `_hub.captureTransaction`, we could technically verify the captured `transaction` and its `measurements`, right? I'm fine keeping this here but ideally, we'd avoid visibility.

Reply:
Do we have access to the transaction instance? Because thats what i'd need.

Reply:
So we cant really mock hub, as we need the real implementation to create `SentryTracer` instance with behaviour and `onFinish` call. Tried some approaches, but only hit road blocks due to implementation details.

- Can't subclass `Hub` to capture transaction, as it only has a factory constructor.
- Can't subclass `SentryClient` to capture transaction, as it only has a factory constructor.
- Mocking the `SentryClient` and capturing the argument does not work.
- Implementing a class that implements `Hub` interface and [delegates](https://stackoverflow.com/a/18569066)  to internal instance does not work, as the tracer gets the delegate instance handed instead of ours.

Keeping measurements accessible though testing or internal annotation is the most straight forward, as the failed approaches would have all tested some other implementation details. WDYT?

Reply:
Let's keep like this then :) thanks for checking that out.